chia/cmds/configure.py

```diff
@@ -59,6 +61,22 @@ def configure(
                 change_made = True
             except ValueError:
                 print("Farmer address must be in format [IP:Port]")
+        if set_solver_peer:
+            try:
+                host, port = parse_host_port(set_solver_peer)
+                # Set single solver peer (overrides any existing)
+                config["farmer"]["solver_peers"] = [{"host": host, "port": port}]
+                print(f"Solver peer updated to {host}:{port}")
+                change_made = True
+            except ValueError:
+                print("Solver address must be in format [IP:Port]")
+        if set_solver_trusted_peers_only:
+            config["solver"]["trusted_peers_only"] = str2bool(set_solver_trusted_peers_only)
```

Comment:
does this work if you upgrade from a previous installation? It seems like the "solver" key may not exist in the config in that case, and this would throw a key error. Or are keys created automatically?

Reply:
you have to have the config section but you have to have it if you run a solver anyway, i am not sure what we usually do but i dont think i want to write the whole section from the code, what do you think ?

Reply:
i can add the sections so the cli commands work but adding all the details about the solver seems wrong, ssl certs path for instance is not added automatically

Reply:
yes, i can do some of the optional stuff automatically but not sure we want to start editing the config for all of the mandatory stuff, we should find out what we did for data-layer since it was also a new service that was added later on